src/main/java/graphql/util/Pair.java

```diff
@@ -0,0 +1,24 @@
+package graphql.util;
+
+import graphql.Internal;
+
+@Internal
+public final class Pair<T, U> {
```

Comment:
A simple Pair class

Reply:
think about guava code instead of home grown